.gitignore

```diff
@@ -1,2 +1,16 @@
+# IntelliJ IDEA
```

Comment:
Do they have to be here? They seem to be a better fitting in your global ignore file.

Reply:
Where is that? I don't know how to configure that. Agreed that better in the global. I am using Idea to write md files.

Reply:
The global gitignore file is at ~/.gitignore_global.